chevah/empirical/mockup.py

```diff
@@ -33,12 +34,21 @@
 
 class _StoppableHTTPServer(BaseHTTPServer.HTTPServer):
     """
-    BaseHTTPServer but with a stoppable server_forever.
+    Single connection HTTP server designed to respond to HTTP requests in
+    functional tests..
     """
+    server_version = 'ChevahTesting/0.1'
+    stopped = False
+    # Current connection served by the server.
+    #active_connection = None
```

Comment:
code  needs to be uncommented... as this is a class member and should be defined here...
left if while trying to replicate a strage bug which occured once on SLES